near-sdk/src/collections/vector.rs

```diff
@@ -52,10 +52,7 @@ impl<T> Vector<T> {
             return None;
         }
         let lookup_key = self.index_to_lookup_key(index);
-        match env::storage_read(&lookup_key) {
-            Some(raw_element) => Some(raw_element),
-            None => env::panic(ERR_INCONSISTENT_STATE),
-        }
+        Some(env::storage_read(&lookup_key).unwrap_or_else(|| env::panic(ERR_INCONSISTENT_STATE)))
```

Comment:
I wonder if, instead of extracting just the message, we should extrct the function itsef:

```
fn expect_storage(x: Option<T>) -> T { x.unwrap_or_else(|| env::panic(""))}
```

not sure what's the best name.

Reply:
that's a good idea, I'll play around with that now (to narrow down inconsistent state errors)

Reply:
Made the change, named it `expect_consistent_state`, let me know if you think that's a good name (I wanted to tie the name to the error message).

I also missed a couple of match statements to collapse, oops!